core/taskengine/vm_test.go

```diff
@@ -230,10 +244,14 @@ func TestRunTaskWithBranchNode(t *testing.T) {
 		},
 	}
 
+	trigger := &avsproto.TaskTrigger{
+		Id:   "triggertest",
+		Name: "triggertest",
+	}
 	edges := []*avsproto.TaskEdge{
 		&avsproto.TaskEdge{
 			Id:     "e1",
-			Source: "__TRIGGER__",
+			Source: trigger.Id,
```

Comment:
A reminder to update the tests/templates of sdk-js to test the real id instead of "__TRIGGER__".